src/main/java/com/faforever/client/chat/ChatUserContextMenuController.java

```diff
@@ -126,6 +128,11 @@ ContextMenu getContextMenu() {
 
   public void setPlayer(Player player) {
     this.player = player;
+    if (showUserInfo.visibleProperty().isBound()) {
+      showUserInfo.visibleProperty().unbind();
+    }
```

Comment:
Do we have to explicitly unbind? Doesn't a bind() call just replace the existing bind?

Reply:
Might be I don't know

Reply:
unbind is not needed...